tools/gulp-extras.js

```diff
@@ -79,8 +79,15 @@ var checkImports = function() {
             var workingDirectory = path.dirname(file.path);
 
             importStatements.forEach(function(importStatement) {
+
                 var modulePath = re.exec(importStatement);
                 if (modulePath && modulePath[1]) {
+
+                    // Module app-center-node-client and it's submodules are relative and doesn't use *.ts files but *.d.ts instead
+                    // so no need to check them
+                    if (modulePath[1].startsWith("../lib/app-center-node-client")) {
```

Comment:
I'd rather not to change the logic of gulp `check-imports` function at all as there is nothing wrong with it at all.
The only thing we should do is to exclude `app-center-node-client` folder.
If we should not check imports here we should better exclude this files from `tsconfig` at all (also I believe these are js files so shouldn't be built in the end?).